third_party/nats-streaming-docker/README.md

```diff
@@ -5,4 +5,4 @@ This is a partial copy of
 current
 [nats-streaming-server](https://github.com/nats-io/nats-streaming-server)
 version is
-[v0.22.1](https://github.com/nats-io/nats-streaming-server/tree/v0.22.1).
+[v0.25.2](https://github.com/nats-io/nats-streaming-server/releases/tag/v0.25.2).
```

Comment:
Thanks for updating this, also please be aware these 3rd party related docs and Dockerfile are just references, in the real installation, the nats stream image is controlled by the controller configmap, where you can specify the image version.

Reply:
Yes, you are right. I saw that after creating the pr. Actually we could drop this third-party part right ? I saw in the helm chart we can pass in a nats container version. We are actually doing that and using now the latest version of nats-streaming-server

Reply:
That's right.